CMakeLists.txt

```diff
@@ -103,6 +103,59 @@ check_stdcxx(${FORCE_CXX})
 
 check_endianness()
 
+###############################################################################
+# Activate Sanitizers
+###############################################################################
+option(SANITIZER "Enable Thread or Address sanitizers" OFF)
+string(TOUPPER ${SANITIZER} SANITIZER)
+
+if(${SANITIZER} STREQUAL "ADDRESS")
```

Comment:
NIT: I thought the idea of having a CMake option `SANITIZER` that allows a string was to avoid writting down cmake code every time a new option want to be used.
Converting `${SANITIZER}` to lowercase and using as string after `-fsanitize` would be enough, wouldn't it?

Reply:
Not addressed as they will be treated differently